sdk/python/build.sh

```diff
@@ -23,7 +23,7 @@
 # Setup:
 #   apt-get update -y
 #   apt-get install --no-install-recommends -y -q default-jdk
-#   wget http://central.maven.org/maven2/io/swagger/swagger-codegen-cli/2.3.1/swagger-codegen-cli-2.3.1.jar -O /tmp/swagger-codegen-cli.jar
```

Comment:
Is the upgrade of swagger version necessary? I thought the 2.3.1 also works for this scenario.

Reply:
There is a problem when i ran the sdk in py 3.7 which is the only py env supported in cloud function. The generated uses async as parameter name which is a keyword in 3.7.